recipe/meta.yaml

```diff
@@ -12,32 +12,32 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: python setup.py build build_ext -I${PREFIX}/include -L${PREFIX}/lib install --single-version-externally-managed --record record.txt # [not win]
-  script: python setup.py install --no-cython --single-version-externally-managed --record record.txt # [win]
+  number: 0
+  script: python setup.py build build_ext -I${PREFIX}/include -L${PREFIX}/lib install --single-version-externally-managed --record record.txt  # [not win]
+  script: python setup.py install --no-cython --single-version-externally-managed --record record.txt  # [win]
+  skip: True
 
 requirements:
   build:
     - python
     - setuptools
-    - toolchain
-    - cython <0.25
+    - cython
     - libev  # [not win]
     - msinttypes  # [win and py2k]
 
   run:
     - python
     - numpy
-    - six >=1.6
-    - futures  # [py2k]
+    - six >=1.9
+    - futures<=2.2.0  # [py2k]
```

Comment:
This needs a single space: `futures <=2.2.0`

Reply:
Done :)